server/src/test/java/org/apache/iotdb/db/integration/IoTDBTestResultSet.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.iotdb.db.integration;
+
+import com.clearspring.analytics.util.Lists;
```

Comment:
Missing apache header, please take a look.

Reply:
and better to changethe name to IoTDBResultSetIT

Reply:
OK